docs/release-1.2/existing-rhdh.md

```diff
@@ -1,15 +1,36 @@
 # Prerequisites
-- RHDH instance deployed with IDP configured (github, gitlab,...)
-- For using the Orchestrator's [software templates](https://github.com/parodos-dev/workflow-software-templates/tree/v1.2.x), OpenShift Gitops (ArgoCD) and OpenShift Pipelines (Tekton) should be installed and configured in RHDH (to enhance the CI/CD plugins)
-- A secret in RHDH's namespace name `dynamic-plugins-npmrc` that points to the plugins npm registry (details will be provided below)
+- RHDH 1.2 instance deployed with IDP configured (github, gitlab, ...)
+- For using the Orchestrator's [software templates](https://github.com/parodos-dev/workflow-software-templates/tree/v1.2.x), OpenShift Gitops (ArgoCD) and OpenShift Pipelines (Tekton) should be installed and configured in RHDH (to enhance the CI/CD plugins) - [Follow these steps](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/docs/gitops/README.md)
```

Comment:
I agree but we don't seem to have that doc in the `release-1.2` branch. We could back port it and update the link though

Reply:
since there isn't any difference between the versions, I don't think we need to duplicate it.
let's wait with duplicating the gitops folder until there is a need for it.